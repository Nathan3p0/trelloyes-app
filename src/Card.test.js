import React from 'react'
import ReactDOM from 'react-dom'
import renderer from 'react-test-renderer'
import Card from './Components/Card'

describe('Card Component Rendering Tests', () => {

    it('renders without crashing', () => {
        const div = document.createElement('div')
        ReactDOM.render(<Card />, div)
        ReactDOM.unmountComponentAtNode(div)
    })

    it('renders to UI as expected', () => {
        const tree = renderer.create(<Card />).toJSON();
        expect(tree).toMatchSnapshot()
    })

    it('renders to UI as expected with PROPS', () => {
        const tree = renderer.create(<Card title="Dinosaurs" content="Died out" />).toJSON();
        expect(tree).toMatchSnapshot()
    })

})