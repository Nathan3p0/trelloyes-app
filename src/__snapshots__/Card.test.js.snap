// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Card Component Rendering Tests renders to UI as expected 1`] = `
<div
  className="Card"
>
  <button
    type="button"
  >
    delete
  </button>
  <h3 />
  <p />
</div>
`;

exports[`Card Component Rendering Tests renders to UI as expected with PROPS 1`] = `
<div
  className="Card"
>
  <button
    type="button"
  >
    delete
  </button>
  <h3>
    Dinosaurs
  </h3>
  <p>
    Died out
  </p>
</div>
`;
